var/spack/repos/builtin/packages/bash/package.py

```diff
@@ -19,6 +19,26 @@ class Bash(AutotoolsPackage):
 
     depends_on('ncurses')
     depends_on('readline@5.0:')
+    depends_on('libiconv')
```

Comment:
I'm not sure if this dependency is actually required, but there doesn't seem to be a way to disable it. Even if you add `--without-libiconv-prefix`, the build still picks up the system library and links to it. There is also an optional `gettext` dependency, but I don't have that on my system, so it doesn't get sucked in. Should we add `gettext` as well? Or what?